test/rssignores.cpp

```diff
@@ -274,6 +274,11 @@ TEST_CASE("RssIgnores::matches() returns true if given RssItem matches any "
 
 			REQUIRE(ignores.matches(&item));
 		}
+
+		SECTION("Throws ConfigHandlerException if second param can't be parsed as filter expression") {
+			REQUIRE_THROWS_AS(ignores.handle_action("ignore-article", {"regex:a{1z", "author = \"John Doe\""}),
+				ConfigHandlerException);
+		}
```

Comment:
Please move this to a test case that tests `handle_action()`:

https://github.com/newsboat/newsboat/blob/4ddbcb50f122c846bb4e766ad03a6e6fcfe00a47/test/rssignores.cpp#L47